ui/src/common/components/CitationSummaryGraph/CitationSummaryGraph.jsx

```diff
@@ -130,6 +267,8 @@ CitationSummaryGraph.propTypes = {
   citeableData: PropTypes.arrayOf(PropTypes.any),
   loadingCitationSummary: PropTypes.bool,
   error: ErrorPropType,
+  onChange: PropTypes.func.isRequired,
```

Comment:
`onQueryChange`

Reply:
To be honest I am also thinking to separate this `query <=> selectedBar` logic to somewhere else so that this `GRAPH` code looks more like an actual graph